README.md

```diff
@@ -2,23 +2,26 @@
 Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+1. Use `brew search` on the command line for a list of packages.
```

Comment:
Agreed to removing `on the command line`. I also don't think `of packages` adds information given the heading.